src/components/Poll.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getPercentage } from '../utils/helpers';
+import { handleAddAnswer } from '../actions/answers';
+
+const getVotesKeys = () => ['aVotes', 'bVotes', 'cVotes', 'dVotes'];
+
+class Poll extends Component {
```

Comment:
can we make this a functional component. Or maybe I dont understand why it needs to be a Class component, mind explaining?

to be more clear I mean that even you would go on and be using classes in this project there is no reason I see for this component to be a class since you not using any `setState` or lifecyrcle methods

Reply:
You are right, there is no reason at all. I mostly stopped using classes after this project. I am going to refactor it completely to practice that and also to make it more "modern" for React.

Reply:
Well this is a good example why the reviewer needs to be the one who resolves the suggestions ;)

I am not suggestng here to refactor to use hooks. But usually is a good practise to use class components when only you use state or lifecyrcles. Here since we agree there is no reason for this to be class component it would be nicer to make it a function ;)

Reply:
For some reason I thought I had to resolve the conversations myself, my bad, I am getting how to flow works little by little.

 And you are right about this case, there is no state in this case so it is not even about using hooks or not.

Reply:
Already refactored Poll.js to be a function instead of a class.
By the way, when I make this comments, should I link the comments to the code I changed?

Reply:
>By the way, when I make this comments, should I link the comments to the code I changed?


good question. No as a reviewer I can see that.